src/super_gradients/recipes/training_hyperparams/default_train_params.yaml

```diff
@@ -7,7 +7,11 @@ warmup_initial_lr: # Initial lr for linear_step. When none is given, initial_lr/
 step_lr_update_freq: # (float) update frequency in epoch units for computing lr_updates when lr_mode=`step`.
 cosine_final_lr_ratio: 0.01 # final learning rate ratio (only relevant when `lr_mode`='cosine')
 warmup_mode: linear_step # learning rate warmup scheme, currently only 'linear_step' is supported
-lr_updates: []
+
```

Comment:
Don't think this file was supposed to be in this PR...

Reply:
This was part of previous PR that is now merged. Now it is removed fro this PR.